.github/workflows/actionlint.yaml

```diff
@@ -46,14 +46,34 @@ jobs:
     # npm ERR! this command with --force or --legacy-peer-deps
     # npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
     - name: wipe_non_yaml_sources
-      run: rm -rf packages/ examples/ extensions/ package.json .github/workflows/test_weaver*.yaml .github/workflows/weaver_deploy*.yml weaver/
+      run: rm -rf packages/ examples/ extensions/ package.json weaver/
+
+    # We nee to exclude these from the linting process for now because these files have 
+    # hudnreds of linter errors that we didn't yet have time to fix. Once the errors are fixed
+    # we can add the files.
```

Comment:
Please fix typos `nee`, `hudnreds`, etc..

Reply:
@outSH Oops, thank you for noticing these! Fixed now!

Reply:
@outSH Could you please take a look at Peter's correction to the commit comment?  This is blocking another PR.